appjs/views/edit_project.js

```diff
@@ -364,6 +445,7 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
             previewUrl = '', iframeLoaded,
             previewSlug = '';
 
+        view.$('#shareText').val(formData['tweet_text']);
```

Comment:
Set the value of the `#shareText` element within the publish tab to contain the share_text that has been saved in the alpaca form after the page is rendered.